druid/datadog_checks/druid/data/conf.yaml.example

```diff
@@ -0,0 +1,204 @@
+init_config:
+    ## @param proxy - object - optional
+    ## Set HTTP or HTTPS proxies for all instances. Use the `no_proxy` list
+    ## to specify hosts that must bypass proxies.
+    ##
+    ## The SOCKS protocol is also supported:
+    ##
+    ## socks5://user:pass@host:port
+    ##
+    ## Using the scheme `socks5` causes the DNS resolution to happen on the
+    ## client, rather than on the proxy server. This is in line with `curl`,
+    ## which uses the scheme to decide whether to do the DNS resolution on
+    ## the client or proxy. If you want to resolve the domains on the proxy
+    ## server, use `socks5h` as the scheme.
+    #
+    # proxy:
+    #   http: http://<PROXY_SERVER_FOR_HTTP>:<PORT>
+    #   https: https://<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    #   no_proxy:
+    #     - <HOSTNAME_1>
+    #     - <HOSTNAME_2>
+
+    ## @param skip_proxy - boolean - optional - default: false
+    ## If set to `true`, this makes the check bypass any proxy
+    ## settings enabled and attempt to reach services directly.
+    #
+    # skip_proxy: false
+
+instances:
+    ## @param url - string - required
+    ## url used to connect to the Druid service.
+    #
+  - url: <DRUID_SERVICE_URL>
+
+    ## @param username - string - optional
+    ## The username to use if services are behind basic auth.
+    #
+    # username: <USERNAME>
+
+    ## @param ntlm_domain - string - optional
+    ## If your services uses NTLM authentication, you can
+    ## specify a domain that is used in the check. For NTLM Auth,
+    ## append the username to domain, not as the `username` parameter.
+    ## Example: <NTLM_DOMAIN>/<USERNAME>
+    #
+    # ntlm_domain: <DOMAIN>
+
+    ## @param password - string - optional
+    ## The password to use if services are behind basic or NTLM auth.
+    #
+    # password: <PASSWORD>
+
+    ## @param connect_timeout - integer - optional
```

Comment:
What's the default value here?  The default value should be used as a placeholder too.

Reply:
fixed